src/icon/mixins.scss

```diff
@@ -95,6 +94,7 @@ $_icon-sizes: (
 }
 
 @mixin style-svg {
+  /* stylelint-disable @cloudscape-design/no-implicit-descendant, selector-max-type, selector-max-universal */
```

Comment:
Is `@cloudscape-design/no-implicit-descendant` needed here?

Reply:
removed and it still passed the linter. Pushed the change